rocksdb/tests/test_options.py

```diff
@@ -67,20 +66,58 @@ def test_compression_opts(self):
         # default value
         self.assertEqual(isinstance(compression_opts, dict), True)
         self.assertEqual(compression_opts['window_bits'], -14)
-        # This doesn't match rocksdb latest
-        # self.assertEqual(compression_opts['level'], -1)
+        self.assertEqual(compression_opts['level'], 2**15 - 1)
         self.assertEqual(compression_opts['strategy'], 0)
         self.assertEqual(compression_opts['max_dict_bytes'], 0)
+        self.assertEqual(compression_opts['zstd_max_train_bytes'], 0)
+        self.assertEqual(compression_opts['parallel_threads'], 1)
+        self.assertEqual(compression_opts['enabled'], False)
 
         with self.assertRaises(TypeError):
-            opts.compression_opts = list(1,2)
+            opts.compression_opts = list(1, 2)
```

Comment:
I'm not sure I understand the purpose of this line. The TypeError exception is raised by just running `list(1, 2)`, so nothing about the options class is being tested.

```python
>>> list(1, 2)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: list expected at most 1 argument, got 2
```

Reply:
Good catch! Let's create an issue to solve this later?